src/configure/utilities/templateConverter.ts

```diff
@@ -22,9 +22,10 @@ export function convertExpression(expression: string): string {
         helperName = expression.substring(openingHelperFunc.length + 1, expression.indexOf(' '));
         parts.push(expression.substring(0, expression.indexOf(' ')) + closingHelperFunc);
         let part = expression.substring(expression.indexOf(' '), expression.indexOf(closingHelperFunc)).trim();
-        let inputs = part.match(parametersRegex) || [];
-        for (let input of inputs) {
-            part = part.replace(input, "{{{" + input + "}}}")
+        let input = parametersRegex.exec(part);
```

Comment:
@anuragc617 can you please review this part

Reply:
@bishal-pdMSFT  yes this looks good. But the template service change will be a breaking change as it will not work for the already released deploy to azure extension because of this bug.

Reply:
Yes, this code needs to go first to prod.

Reply:
Yes, thats true. We need to be careful here. @ashwinsangem lets sync up how we can deal with this. I guess we need versioning sooner than later. But lets plan what we can do in short term.

Reply:
BTW @ashwinsangem I believe this PR should go in regardless of other things.